### PR TITLE
feat: add `allowCross` property #56

### PR DESCRIPTION
**Pull request by benjycui**
Close: #56

### CODE REVIEW

**Comment by just-boris** [comment]
Looks good. Does it need to add into `propTypes` section too?

File: src/Slider.jsx [hunk 1]
```diff
@@ -401,6 +419,7 @@ Slider.defaultProps = {
   disabled: false,
   dots: false,
   range: false,
+  allowCross: false,
```
Comment (afc163) [inline src/Slider.jsx]: default true?

**Comment by benjycui** [comment]
@just-boris @afc163 done

**Comment by benjycui** [comment]
Merge now.

**Pull request merged**